modules/webbrowser/properties/propertywidgetcef.cpp

```diff
@@ -43,7 +79,13 @@ void PropertyWidgetCEF::setFrame(CefRefPtr<CefFrame> frame) {
     frame_ = frame;
```

Comment:
This should call setIfPartOfFrame?

Reply:
Changed so that setFrame() calls setIfPartOfFrame instead of the previous above.

widget->setFrame(frame) is then called istead of widget->setIfPartOfFrame(frame) from the class that synchronizes all widgets (propertycefsynchronizer.cpp).